src/components/WorkshopPage/WorkshopPage.js

```diff
@@ -277,7 +278,12 @@ function WorkshopPage() {
 					<Typography variant='h4' component='h1' className={classes.title}>
 						Workshops
 					</Typography>
-					{workshopCards}
+					{!workshopsAvailable ?
+						<ComingSoon alignment='left' /> :
+						<div>
+							{workshopCards}
```

Comment:
Is your linter throwing errors for this line? Mine does and wants spaces around the curly braces, like `{ workshopCards }`